clients/devices.js

```diff
@@ -3,11 +3,13 @@
 module.exports = {
   name: 'Is the devices list settings section visible',
   startDate: '2015-01-01',
-  subjectAttributes: ['forceDeviceList'],
+  subjectAttributes: ['forceDeviceList', 'isMetricsEnabledValue'],
```

Comment:
Does this need `uniqueUserId` similar to [mailcheck](https://github.com/mozilla/fxa-content-experiments/blob/dev/verification/mailcheck.js#L7)?